### PR TITLE
OSDOCS-10365: 4.14.23 RN

### PR DESCRIPTION
**Pull request by obrown1205**
Version(s):
4.14

Issue:
[OSDOCS-10365](https://issues.redhat.com/browse/OSDOCS-10365)

Link to docs preview:
https://75348--ocpdocs-pr.netlify.app/openshift-enterprise/latest/release_notes/ocp-4-14-release-notes.html#ocp-4-14-23

QE review:
N/A

Additional information:

### CODE REVIEW

File: release_notes/ocp-4-14-release-notes.adoc [hunk 1]
```diff
@@ -3318,6 +3318,63 @@ This section will continue to be updated over time to provide notes on enhanceme
...
+
+* Previously, there was a fundamental issue that prevented egress IPv6 from working for Elastic IPs (EIPs) with multi-network interface controllers (NICs). With this release, the issue has been resolved. (link:https://issues.redhat.com/browse/OCPBUGS-31853[*OCPBUGS-31853*])
+
+* Previously, the `kube-rbac-proxy-crio` label had mistyped resources. With this release, the issue has been resolved. (link:https://issues.redhat.com/browse/OCPBUGS-32260[*OCPBUGS-32260*])
+
+* Previously, `v4-transit-switch-subnet` and `v6-transit-switch-subnet` flags were added, but some configurations were still missing to allow setting custom subnets while creating a kind cluster. With this release, it is possible to override default v4 and v6 transit switch subnets while creating a kind cluster. (link:https://issues.redhat.com/browse/OCPBUGS-32353[*OCPBUGS-32353*])
```
Comment (mburke5678) [inline release_notes/ocp-4-14-release-notes.adoc:3367]: > some configurations were still missing to allow setting custom subnets 
This seems to be missing something. Not sure if this is correct. 
```suggestion
* Previously, `v4-transit-switch-subnet` and `v6-transit-switch-subnet` flags were added, but some configurations that allow for setting custom subnets while creating a kind cluster were still missing. With this release, it is possible to override default v4 and v6 transit switch subnets while creating a kind cluster. (link:https://issues.redhat.com/browse/OCPBUGS-32353[*OCPBUGS-32353*])
```

Comment (mburke5678) [reply release_notes/ocp-4-14-release-notes.adoc:3367]: Does a typical user know what a _kind cluster_ is?

File: release_notes/ocp-4-14-release-notes.adoc [hunk 2]
```diff
@@ -3318,6 +3318,63 @@ This section will continue to be updated over time to provide notes on enhanceme
...
+
+* Previously, certain HTTP clients caused Ingress traffic to degrade after upgrading to {product-title} 4.14 when closed idle connections were erroneously reused. With this release, the issue has been resolved. (link:https://issues.redhat.com/browse/OCPBUGS-32437[*OCPBUGS-32437*])
+
+* Previously, the image registry's Azure path fix job incorrectly required the presence of client and tenant IDs to function, which resulted in valid configurations that threw validation errors. With this release, a check to account for key-in connection to missing client and tenant IDs is added. (link:https://issues.redhat.com/browse/OCPBUGS-32450[*OCPBUGS-32450*])
```
Comment (mburke5678) [inline release_notes/ocp-4-14-release-notes.adoc:3371]: Per ISG, use _throw_ only with Java docs.
```suggestion
* Previously, the image registry's Azure path fix job incorrectly required the presence of client and tenant IDs to function, which caused valid configurations to produce validation errors. With this release, a check to account for key-in connection to missing client and tenant IDs is added. (link:https://issues.redhat.com/browse/OCPBUGS-32450[*OCPBUGS-32450*])
```
```suggestion
* Previously, the image registry's Azure path fix job incorrectly required the presence of client and tenant IDs to function, which caused valid configurations to fail validation. With this release, a check to account for key-in connection to missing client and tenant IDs is added. (link:https://issues.redhat.com/browse/OCPBUGS-32450[*OCPBUGS-32450*])
```

**Comment by mburke5678** [comment]
@obrown1205 A few nits. Otherwise LGTM

File: release_notes/ocp-4-14-release-notes.adoc (hunk 1 above)
Comment (obrown1205) [reply release_notes/ocp-4-14-release-notes.adoc:3367]: Good question - confirming with dev

Comment (obrown1205) [reply release_notes/ocp-4-14-release-notes.adoc]: Removing this bug - this feature has two parts, ovn-kubernetes side of things and cluster network operator side of things. Dev has finished backporting ovn-kubernetes part to 4.14 but cluster-b=network-operator side of things have not yet made it to 4.14.

**Pull request merged**